.eslintrc.js

```diff
@@ -123,6 +123,14 @@ module.exports = {
 						importNames: [ 'combineReducers' ],
 						message: "`combineReducers` should be imported from 'state/utils', not 'redux'.",
 					},
+					// Deprecate createReducer and createReducerWithValidation.
+					{
+						name: 'state/utils',
```

Comment:
I searched and don't see any imports from `state/utils/create-reducer[-with-validation]` so this should cover us 👍